facebook/delphiFacebook/R/binary.R

```diff
@@ -135,6 +135,26 @@ get_binary_indicators <- function() {
     "smoothed_try_vaccinate_1m", "weight_unif", "v_try_vaccinate_1m", 6, compute_binary_response, jeffreys_binary,
     "smoothed_wtry_vaccinate_1m", "weight", "v_try_vaccinate_1m", 6, compute_binary_response, jeffreys_binary,
     
+    "smoothed_winitial_dose_one_of_one", "weight", "v_initial_dose_one_of_one", 6, compute_binary_response, jeffreys_multinomial_factory(3),
+    "smoothed_winitial_dose_one_of_two", "weight", "v_initial_dose_one_of_two", 6, compute_binary_response, jeffreys_multinomial_factory(3),
+    "smoothed_winitial_dose_two_of_two", "weight", "v_initial_dose_two_of_two", 6, compute_binary_response, jeffreys_multinomial_factory(3),
+    
+    "smoothed_wvaccinated_one_booster", "weight", "v_vaccinated_one_booster", 6, compute_binary_response, jeffreys_multinomial_factory(3),
+    "smoothed_wvaccinated_two_or_more_boosters", "weight", "v_vaccinated_two_or_more_boosters", 6, compute_binary_response, jeffreys_multinomial_factory(3),
+    "smoothed_wvaccinated_no_booster", "weight", "v_vaccinated_no_booster", 6, compute_binary_response, jeffreys_multinomial_factory(3),
+    "smoothed_wvaccinated_at_least_one_booster", "weight", "v_vaccinated_at_least_one_booster", 6, compute_binary_response, jeffreys_binary,
+    
+    "smoothed_wvaccinated_booster_accept", "weight", "v_vaccinated_booster_accept", 6, compute_binary_response, jeffreys_binary,
+    "smoothed_wvaccinated_booster_hesitant", "weight", "v_vaccinated_booster_hesitant", 6, compute_binary_response, jeffreys_binary,
+    
+    "smoothed_wvaccinated_booster_defyes", "weight", "v_vaccinated_booster_defyes", 6, compute_binary_response, jeffreys_multinomial_factory(4),
+    "smoothed_wvaccinated_booster_probyes", "weight", "v_vaccinated_booster_probyes", 6, compute_binary_response, jeffreys_multinomial_factory(4),
+    "smoothed_wvaccinated_booster_probno", "weight", "v_vaccinated_booster_probno", 6, compute_binary_response, jeffreys_multinomial_factory(4),
+    "smoothed_wvaccinated_booster_defno", "weight", "v_vaccinated_booster_defno", 6, compute_binary_response, jeffreys_multinomial_factory(4),
+    
+    "smoothed_wflu_vaccine_2021", "weight", "v_flu_vaccine_2021", 6, compute_binary_response, jeffreys_binary,
```

Comment:
I'd suggest `smoothed_wflu_vaccinated_2021`, to be consistent with `covid_vaccinated`, but not a big deal if Facebook prefers one name or the other

Reply:
Checking with FB.

Reply:
They're fine with changing this (there's no matching signal on UMD's side).